tests/basic_binary_operation.test.ts

```diff
@@ -18,7 +18,7 @@ describe('Basic return value test', () => {
         const result = executeGraph(g);
         expect(result.returnValue()).toBe(3);
     });
-
+    /*
```

Comment:
Regarding this and the other tests - i think we should not drop test cases because of that async behavior technicality.
Im sure we can make jest behave synchronously, i can take this task if you wish.